lib/db.js

```diff
@@ -11,6 +11,7 @@ function initDb(dbUrl, callback){ // eslint-disable-line
     MongoClient.connect(dbUrl, { useNewUrlParser: true, useUnifiedTopology: true }, connected);
     function connected(err, client){
         if(err){
+            console.log('Failed connecting to the DB', err);
```

Comment:
Was this left by mistake, or there is a use of it in production?

Reply:
I added deliberately as the app fails silently when a DB connection cannot be made.